### DISCUSSION TITLE
Web3Auth X Kanji

### DISCUSSION THREAD

**Discussion by ghost** [Web3Auth/Web3Auth#1215] [Help]
Hello web3auth, 

We have few questions because we want to integrate your product in Kanji. Notably around the integration of a login/pw of a brand (basically if a user has a profile in the crm of the brand he can use the same given that we have access to the api of the brand). 

Can we get connected with the right person ? 

What is Kanji? 
Kanji is a no-code platform to launch NFT projects in a few clicks. 

With Kanji, brands can master the NFT tech and market to build and entertain their fan base. Drive loyalty, boost lifetime value and fully leverage the power of community-based marketing. Create unique fan experiences that can only be unlocked via NFTs: exclusive events, merchandise, online community access, and more.

**Comment by Saviour1001** [comment]
Hey there, you can have a look at the Enterprise support features we provide on our [pricing page](https://web3auth.io/pricing.html). On the same page, you can contact the team to help you out.
